apps/aecore/src/aec_dry_run.erl

```diff
@@ -96,20 +96,23 @@ dry_run_int([], _Trees, Env, _Opts, Acc) ->
 dry_run_int([{tx, TxOpts, Tx} | Txs], Trees, Env, Opts, Acc) ->
     Stateless = proplists:get_value(stateless, TxOpts, false),
     Env1 = prepare_env(Env, TxOpts),
+    EventsEnabled = proplists:get_bool(tx_events, Opts),
     %% GH3283: Here we should collect and present the `internal_call_tx` events.
     %% This means expanding the return type, and breaking the api :scream_cat:.
```

Comment:
Possibly remove this comment, since we do this, and break the api 😛

Reply:
Comment removed